crontab.js

```diff
@@ -99,7 +106,7 @@ exports.set_crontab = function(env_vars, callback){
 				}
 
 				if (tab.mailing && JSON.stringify(tab.mailing) != "{}"){
-					crontab_string += "; /usr/local/bin/node " + __dirname + "/bin/crontab-ui-mailer.js " + tab._id + " " + stdout + " " + stderr;
```

Comment:
I'm not sure why I made that change now.   It may have been an accident.   I'll give it a test in my setup, and put it back if I can't figure out why I did that.   Thanks